mmcv/utils/env.py

```diff
@@ -56,16 +56,20 @@ def collect_env():
         if CUDA_HOME is not None and osp.isdir(CUDA_HOME):
             try:
                 nvcc = osp.join(CUDA_HOME, 'bin/nvcc')
-                nvcc = subprocess.check_output(
-                    f'"{nvcc}" -V | tail -n1', shell=True)
+                nvcc = subprocess.check_output(f'"{nvcc}" -V', shell=True)
```

Comment:
It could be better if we can get the last line of `nvcc -V` which is consistent with the former version.

Reply:
On Linux
```
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2019 NVIDIA Corporation
Built on Wed_Oct_23_19:24:38_PDT_2019
Cuda compilation tools, release 10.2, V10.2.89
```

On Windows (11.5)
```
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2021 NVIDIA Corporation
Built on Thu_Nov_18_09:52:33_Pacific_Standard_Time_2021
Cuda compilation tools, release 11.5, V11.5.119
Build cuda_11.5.r11.5/compiler.30672275_0
```

Not sure what it looks like on other Windows machines. but the last line on Linux match with the second last line on Windows. 

I will add 'Cuda compilation tools' to the output.

Reply:
OK~ thanks for your explanation.

Reply:
May leave comments in the code to explain.